assets/js/blocks/cart/cart-cross-sells-product-list/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { ProductResponseItem } from '@woocommerce/type-defs/product-response';
+
+/**
+ * Internal dependencies
+ */
+import CartCrossSellsProduct from './cart-cross-sells-product';
+
+const placeholderRows = [ ...Array( 3 ) ].map( ( _x, i ) => (
+	<CartCrossSellsProduct crossSellsProduct={ {} } key={ i } />
+) );
+
+interface CrossSellsProductListProps {
+	crossSellsProducts: ProductResponseItem[];
+	isLoading: boolean;
+	className?: string;
+	columns: number;
+}
+
+const CartCrossSellsProductList = ( {
+	crossSellsProducts,
+	isLoading = false,
```

Comment:
I never saw this be `true`, could it be because our block is only rendered _after_ the initial cart loading work is complete?

Reply:
That's possible, and I removed this part as a result of https://github.com/woocommerce/woocommerce-blocks/pull/6645/files/aed95be381fedeb379eabc460a78e591b8106a61#r956144991.